v-next/hardhat/src/internal/cli/telemetry/sentry/anonymizer.ts

```diff
@@ -17,13 +17,16 @@ import * as korean from "ethereum-cryptography/bip39/wordlists/korean.js";
 import * as simplifiedChinese from "ethereum-cryptography/bip39/wordlists/simplified-chinese.js";
 import * as SPANISH from "ethereum-cryptography/bip39/wordlists/spanish.js";
 import * as traditionalChinese from "ethereum-cryptography/bip39/wordlists/traditional-chinese.js";
-import { either } from "fp-ts";
 
 interface WordMatch {
   index: number;
   word: string;
 }
 
+type AnonymizeResult =
```

Comment:
We should export this type as its part of the Anonymizer API.

Reply:
Done